test/Microsoft.AspNet.Razor.Test/Framework/ParserTestBase.cs

```diff
@@ -336,7 +337,14 @@ private static void EvaluateSyntaxTreeNode(ErrorCollector collector, SyntaxTreeN
                 collector.AddMessage("{0} - PASSED :: Attribute names match", expected.Key);
             }
 
-            EvaluateSyntaxTreeNode(collector, actual.Value, expected.Value);
+            if (actual.Value == null && expected.Value == null)
+            {
+                collector.AddMessage("{0} - PASSED :: Attribute values match", expected.Key);
```

Comment:
make message specific to minimized case and don't mention "values" e.g. `Minimized attributes match`.